Majasdarbs/03-majasdarbs/AndzejsDargelis/CV/CV-index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>CV-Andžejs-Darģelis</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <label class="dropdown" for="toggle">&#9776;</label>
+      <input type="checkbox" id="toggle">
```

Comment:
in future probably make more unique ids. There could me multiple toggles across the app. Could be toggle-married or smth like that.